content/applications/websites/website/web_design/themes.rst

```diff
@@ -0,0 +1,84 @@
+  :guilabel:`Background`, :guilabel:`Text`, :guilabel:`Headings`, :guilabel:`Links`,
+  :guilabel:`Primary Buttons`, and :guilabel:`Secondary Buttons`.
+
+  .. image:: themes/colors.png
+     :alt: select the colors of your website
+
+  **To apply a color preset** to a building block of your site, select it, go to the
+  :guilabel:`Customize` tab, click the :guilabel:`Background` button and select the one you want.
+  If you modify later a color preset that have been applied to some elements of your site, they will
+  all be updated automatically.
+
+- :guilabel:`Website`: From this section, you can :guilabel:`Switch Theme`,
+  :guilabel:`Add a Language`, select the :guilabel:`Page Layout`, and customize the
+  :guilabel:`Background` by uploading your own image.
+
+  .. seealso::
+     :doc:`Translations <../configuration/translate>`
+
+- :guilabel:`Paragraph`: In this section you can customize the format of the text <p> of your
+  website.
+
+  .. tip::
+     The :guilabel:`Font Family` field contains fonts that are hosted and served by Google servers.
+     To add another font, click :guilabel:`Add a Google Font`, and in the pop up window, click
+     :guilabel:`fonts.google.com`.
+
+     .. image:: themes/add-a-font.png
+        :alt: select the font you like
+
+     Select a font you like, copy the address of the page and paste it in the :guilabel:`Google Font
+     address` field, then :guilabel:`Save and Reload`. The new font applies to your entire website.
+
+- :guilabel:`Headings`: In this section you can customize the format of your headings.
+
+- :guilabel:`Button`: Two types of buttons exist in Odoo, the :guilabel:`Primary Style` and the
+  :guilabel:`Secondary Style`. You can update their style as per your preference from this section.
+
+  .. image:: themes/buttons.png
+     :alt: two types of button in Odoo
+
+- :guilabel:`Link`: This section allows you to edit the style of the hyperlinks available on your
+  website.
+
+- :guilabel:`Input Fields`: An input field is a field where you can enter data or provide input,
+  e.g., a search bar or a form. You can customize these fields from this section.
+
+- :guilabel:`Advanced`: You can decide whether or not to hide the header bar of your website using
+  the :guilabel:`Show Header` button, inject :guilabel:`<head> and </body>` code, enter your
+  Google map custom key, change the colors of the :guilabel:`Success`, :guilabel:`Info`,
+  :guilabel:`Warning`, :guilabel:`Error` pop up messages by clicking the related
+  :guilabel:`Status Colors` buttons, and customize the :guilabel:`Grays` elements of your site.
+
+  .. example::
+     The :guilabel:`Status Color` of the :guilabel:`Success` messages is set to green.
+
+     .. image:: themes/advanced.png
+        :alt: Status colors success message set to green.
+
+     .. image:: themes/success.png
+        :alt: Success message is green.
+
+  .. example::
+     Customize the grays elements of your site.
```

Comment:
```suggestion
     Customizing the grays elements of your site.
```
I believe it should be "customizing" since we're showing what's being done, not giving an instruction "customize this by doing XYZ"